Email Services/CreateContactFromEmail.cls

```diff
@@ -0,0 +1,69 @@
+global class CreateContactFromEmail implements Messaging.InboundEmailHandler {
+    global Messaging.InboundEmailResult handleInboundEmail(Messaging.inboundEmail email, Messaging.InboundEnvelope env){
+        Messaging.InboundEmailResult result = new Messaging.InboundEmailResult();
+        try{
+            if(email.textAttachments.size() > 0 && email.textAttachments[0].filename.endsWith('.xml'))
+            {
+                String myPlainText= '';
+                myPlainText = email.plainTextBody;
+                List<Contact> contactList = new List<Contact>();
+                if(email.textAttachments != null)

```

Comment:
we are already checking the size at line 5, is it possible that we will get null here ?

Reply:
Ok I'll Remove it